app/gameengine.py

```diff
@@ -3450,6 +3553,20 @@ def archive_hand_continuation():
                 effect_copy["amount"] *= num_of_stacked_cards
                 effect_copy["effect_type"] = EffectType.EffectType_DealDamage
                 self.add_effects_to_front([effect_copy])
+            case EffectType.EffectType_DealLifeDamage:
+                amount = effect["amount"]
+                opponent = effect.get("opponent", False)
+                source_player = self.get_player(effect_player_id)
+                target_player = self.other_player(effect_player_id) if opponent else effect_player
+                source_holomem_card, _, _ = source_player.find_card(effect["source_card_id"])
+                if not source_holomem_card:
+                    # Assume this is an attachment, find it on the holomem.
+                    for holomem in source_player.get_holomem_on_stage():
+                        for attachment in holomem["attached_support"]:
+                            if attachment["game_card_id"] == effect["source_card_id"]:
+                                source_holomem_card = holomem
+                                break
+                self.deal_life_damage(target_player, source_holomem_card, amount, self.continue_resolving_effects)
```

Comment:
i think we need to set passed_on_continuation here because this will trigger a decision and we don't want both that and this to continue resolving effects.